miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -856,10 +859,23 @@ protected void writeConfiguration(ObjectMapper mapper, ObjectNode config) throws
       }
       config.put("create", true);
       config.put("hasProject", project != null);
+      Map<Long, List<LibraryTemplateDto>> templatesByProjectId = new HashMap<>();
       if (project == null) {
         projectService.listAllProjects().stream().map(Dtos::asDto).forEach(config.putArray("projects")::addPOJO);
+        List<LibraryTemplate> templates = libraryTemplateService.list();
+        for (LibraryTemplate template : templates) {
+          LibraryTemplateDto dto = Dtos.asDto(template);
+          for (Project project : template.getProjects()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "project" which hides the field declared at line 837. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)